src/autosuggest/interfaces.ts

```diff
@@ -75,7 +75,7 @@ export interface AutosuggestProps
   /**
    * Specifies a function that generates the custom value indicator (for example, `Use "${value}"`).
    */
-  enteredTextLabel: AutosuggestProps.EnteredTextLabel;
+  enteredTextLabel?: AutosuggestProps.EnteredTextLabel;
```

Comment:
The website API documentation will state `enteredTextLabel ` as `optional` in this case? But it is actually required value. As enteredTextLabel item is always shown, if the user does not set it, it will display an empty row. I am thinking maybe make it clear in the documentation, always provide it unless using i18n privider?